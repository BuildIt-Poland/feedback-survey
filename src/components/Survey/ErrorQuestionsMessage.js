import React from 'react';
import { string } from 'prop-types';
import { Box, Heading } from 'grommet';

import { colorRed } from '../../styles/variables';

const ErrorQuestionsMessage = ({ message }) => (
  <Box align="center" justify="center" pad="medium" height="90vh">
    <Heading level="4" textAlign="center" color={colorRed}>
      {message}
    </Heading>
  </Box>
);

ErrorQuestionsMessage.propTypes = {
  message: string.isRequired
};

export default ErrorQuestionsMessage;
