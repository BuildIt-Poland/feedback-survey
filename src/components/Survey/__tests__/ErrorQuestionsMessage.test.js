import React from 'react';
import { create } from 'react-test-renderer';

import ErrorQuestionsMessage from '../ErrorQuestionsMessage';

describe('COMPONENT - ErrorQuestionsMessage', () => {
  it('renders correctly', () => {
    const message = 'Test error message';
    const component = create(<ErrorQuestionsMessage message={message} />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
