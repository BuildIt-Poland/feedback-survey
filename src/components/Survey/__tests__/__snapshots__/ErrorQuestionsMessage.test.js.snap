// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - ErrorQuestionsMessage renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 ePVEiU"
>
  <h4
    className="StyledHeading-sc-1rdh4aw-0 btqNMh"
  >
    Test error message
  </h4>
</div>
`;
